user/pmbootstrap/patches/mknod.patch

```diff
@@ -0,0 +1,15 @@
+--- a/pmb/chroot/mount.py	2024-05-18 13:34:09.000000000 -0300
++++ b/pmb/chroot/mount.py	2024-09-25 12:37:10.157484531 -0300
+@@ -20,11 +20,11 @@
+             path = chroot + "/dev/" + str(dev[4])
+             if not os.path.exists(path):
+                 pmb.helpers.run.root(args, ["mknod",
+-                                            "-m", str(dev[0]),  # permissions
+                                             path,  # name
+                                             str(dev[1]),  # type
+                                             str(dev[2]),  # major
+                                             str(dev[3]),  # minor
++                                            "root:root"
```

Comment:
this changes from `dev[0]` to hardcoded root:root, doesn't seem correct?

Reply:
i just tested this out and it seemed to fix all of the issues i was getting with mknod, it just hardcodes root:root to be the owner of the devices that it makes, instead of setting `0666` perms - setting this hardcoded doesnt seem like the best idea tho.

Reply:
the correct thing is to merely do

```
pmb.helpers.run.root(args, ["mknod"], ....)
pmb.helpers.run.root(args, ["chmod"], path, something)
```

or whatnot.

or iow: you want https://gitlab.com/postmarketOS/pmbootstrap/-/commit/375814b9d2f0d08d221093e8f9d82c50d2f234c4

Reply:
fair!